org.opent2t.sample.multisensor.superpopular/com.wink.sensorpod/js/thingTranslator.js

```diff
@@ -84,7 +89,7 @@ function convertDeviceBatteryToTranslatorBattery(batteryValue) {
  * Returns a default value if the specified property is null, undefined, or an empty string
  */
 function defaultValueIfEmpty(property, defaultValue) {
-    if (property === undefined || property === null || property === "") {
+    if (!property) {
```

Comment:
I see -- ok will revert here. 

---
In reply to: [106045877](https://github.com/openT2T/translators/pull/210#discussion_r106045877) [](ancestors = 106045877)